todo/src/Reducers/rootReducer.js

```diff
@@ -0,0 +1,11 @@
+import toggleReducer from './toggleReducer'
+import todoReducer from './todoReducer'
+
+import { combineReducers } from 'redux';
+
+const rootReducer = combineReducers({
```

Comment:
Think of the rootReduce as a function that takes in other functions as values of an object. where the keys are the state,
So a typical reducer would be an object where each key is a different state.
{
user:userReducer,
comments: commentsReducer,
posts:postReducer
}